corehq/ex-submodules/pillowtop/es_utils.py

```diff
@@ -22,6 +22,10 @@ def _get_analysis(*names):
     "comma": {
         "type": "pattern",
         "pattern": r"\s*,\s*"
+    },
+    "phonetic_analyzer": {
```

Comment:
nit: drop the 'analyzer' suffix
```suggestion
    "phonetic": {
```

Reply:
some could be done for `soundex_filter` -> `soundex`